tensorflow-tools/src/main/java/org/tensorflow/tools/ndarray/impl/sequence/SingleElementSequence.java

```diff
@@ -24,7 +24,11 @@
 import org.tensorflow.tools.ndarray.NdArraySequence;
 import org.tensorflow.tools.ndarray.impl.AbstractNdArray;
 
-class SingleElementSequence<T, U extends NdArray<T>> implements NdArraySequence<U> {
+public final class SingleElementSequence<T, U extends NdArray<T>> implements NdArraySequence<U> {
```

Comment:
Class level javadoc, now it's public? Or is this something we'd exclude using modules if we were on Java 9?

Reply:
yes that would be something we would exclude in Java 9, pretty much everything under `impl` in fact should be limited to internal usage only. Still, some javadoc wouldn't hurt.